.github/workflows/release.yml

```diff
@@ -1,24 +1,34 @@
+# This workflow should only be used for version releases for cadenza.js versions bundled with
+# Cadenza 10.2 and later. Starting with 10.2 we do not use Semantic versioning in the classic way any longer.
+# For earlier versions use the "Legacy Release" workflow.
+
 name: Release
 
 on:
   workflow_dispatch:
     inputs:
-      release-type:
-        type: choice
-        description: 'Release type'
-        required: true
-        options:
-          - patch
-          - minor
-          - major
-        default: 'patch'
+      cadenza-version:
+        type: string
+        description: |
+          Cadenza Version
```

Comment:
Naming Proposal: Cadenza Main Version

That's how it's called in the Maven_Parent/pom.xml

Reply:
Updated.